tests/topotests/ospf_flexalgo_topo1/rt1/ospfd_flxalg.conf

```diff
@@ -0,0 +1,37 @@
+password 1
+hostname rt1
+log file ospfd.log
+log syslog debugging
+!
+debug ospf sr
+! debug ospf te
+! debug ospf event
+! debug ospf lsa
+debug ospf lsa generate
+! debug ospf zebra
+!
+interface lo
+ ip ospf area 0.0.0.0
+!
+interface eth-rt1-1
+ ip ospf area 0.0.0.0
+!
+interface eth-rt1-2
+ ip ospf area 0.0.0.0
+!
+router ospf
+ ospf router-id 1.1.1.1
+ capability opaque
+ router-info area 0.0.0.0
+ flexible-algorithm 128 metric-type igp
+ flexible-algorithm 128 calculation-type spf
+ flexible-algorithm 128 priority 10
+ flexible-algorithm 128 exclude-admin-group 30
+ flexible-algorithm 128 include-all-admin-group 20
+ flexible-algorithm 128 include-any-admin-group 60
+ flexible-algorithm 128 include-any-admin-group 70
+ flexible-algorithm 128 exclude-srlg 10
+ flexible-algorithm 128 flags 10
+ flexible-algorithm 128 prefix-advertise-metric 100
```

Comment:
For consistency with IS-IS, could you create a flex-algo context and make sure the commands are consistent?

https://github.com/FRRouting/frr/blob/master/tests/topotests/isis_sr_flex_algo_topo1/rt1/isisd.conf

You should have a command to say if the router is advertising the definition